repos/fdbt-site/src/pages/api/defineCaps.ts

```diff
@@ -0,0 +1,72 @@
+import { NextApiResponse } from 'next';
+import { CAPS_DEFINITION_ATTRIBUTE } from '../../constants/attributes';
+import { getCapByNocAndId } from '../../data/auroradb';
+import { getAndValidateNoc, redirectTo, redirectToError } from '../../utils/apiUtils/index';
+import { CapInfo, CapsDefinitionWithErrors, NextApiRequestWithSession } from 'src/interfaces';
+import { updateSessionAttribute } from '../../../src/utils/sessions';
+
+const getCapContent = async (cap: number | undefined, noc: string): Promise<CapInfo | undefined> => {
+    if (!cap) {
```

Comment:
I think this function should take (capId: number) as the argument, and only be called if !!capId